scopes/react/ui/docs-app/index.tsx

```diff
@@ -2,9 +2,7 @@ export { DocsTheme } from './docs-theme';
 export type { DocsAppProps } from './docs-theme';
 
 export { default } from './docs.app-root';
-export type { ReactDocsRoot } from './docs.app-root';
 
-export { defaultDocs } from './docs-app';
 export type { ReactDocsAppProps } from './docs-app';
 
-export {default as docsStyles} from './docs-app.module.scss';
+export { default as docsStyles } from './docs-app.module.scss';
```

Comment:
please expose these styles explicitly. A little more work but a much better api

```ts
// docs-app.styles.ts
export const classes = {
  compositionSection: docsStyles.compositionSection
  compositionCard: docsStyles.compositionCard,
  docsMainBlock: docsStyles.docsMainBlock,
  mdx: docsStyles.mdx,
}
```

Reply:
this isnt part of this PR - we can do a lot more with the code but not in the scope

Reply:
ahh I see